daemon/config/config_windows.go

```diff
@@ -8,8 +8,16 @@ const (
 	// This is used by the `default-runtime` flag in dockerd as the default value.
 	// On windows we'd prefer to keep this empty so the value is auto-detected based on other options.
 	StockRuntimeName = ""
```

Comment:
Looking at this bit again, and wondering if we should continue "not using" this for Windows, or if we should put `com.docker.hcsshim.v1` here (default (for now) to be the built-in runtime, and `io.containerd.runhcs.v1` to be the "opt in" for containerd). Where at some point we can change the default.

Reply:
Comment was added in https://github.com/moby/moby/commit/7ccf750daad37aa12e790d5f057b7ae6a140e0b8 (https://github.com/moby/moby/pull/42089) ./cc @cpuguy83 perhaps you recall the details on this 🤔